docs/version_db_auth.md

```diff
@@ -8,21 +8,29 @@
 |--------------------|--------------|------------|-----|------|---------|-------|---------|
 | [sql_rev][1]       | VARCHAR(100) |            | PRI | NO   |         |       |         |
 | [required_rev][2]  | VARCHAR(100) |            | MUL | YES  |         |       |         |
-| [2020_02_07_00][3] | bit(1)       |            |     | YES  |         |       |         |
+| [date][3]          | VARCHAR(50)  |            |     | YES  |         |       |         |
+| [2021_06_17_00][4] | BIT(1)       |            |     | YES  |         |       |         |
 
 [1]: #sql_rev
 [2]: #required_rev
 [3]: #2020_02_07_00
+[4]: #2021_06_17_00
```

Comment:
```suggestion
[1]: #sqlrev
[2]: #requiredrev
[3]: #date
[4]: #2021061700
```

Reply:
_ doesnt work in links, so you just cut them from the link and it will work :D

Reply:
it didn't work like that https://github.com/azerothcore/wiki/pull/573/commits/a036b578f40be0e76678d59ae01afdf45991fd13 

so if it worked for me https://github.com/azerothcore/wiki/pull/573/commits/4544705eed454997bc0c19728ed371dbd0a97af2